packages/typespec-ts/test/integration/generated/authentication/api-key/auth-apikey.api.md

```diff
@@ -0,0 +1,105 @@
+## API Report File for "@msinternal/auth-apikey"
```

Comment:
Instead of the api.md we should keep the rollup.d.ts which will allow us to also reference the client in tests without the implementation being present. tspconfig.yaml needs to be present as well as things are organized today.